src/com/codersbay/PersonManagment.java

```diff
@@ -0,0 +1,44 @@
+package com.codersbay;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class PersonManagment {
+
+    public static ArrayList<Person> allPersons = new ArrayList<>();
+
+    public static void addPerson(Person p) {
```

Comment:
Hier war die Angabe der übung leider nicht so schön geschrieben sorry - eigentlich geht's darum dass die PersonenVerwaltung die Klasse ist, die Personen für dich erstellt. Das heißt addPerson sollte eine Person zurückgeben und dafür in der parameterliste die argumente für die personen akzeptieren - also Name, Geburtsdatum, Adresse, ... 

Im besten fall übst du hier auch ein bisschen das method overloading und machst verschiedene `addPerson` methoden auf der `PersonManagement` Klasse.

Reply:
ich glaube was du meinst ist bei mir die "createPerson" Methode. Mit "addPerson" füge ich diese dann nur zu der Liste hinzu, weil ich ein bisschen mit Methoden arbeiten wollte.